get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+﻿---

```

Comment:
I think this file location needs to be changed. 😄 

And help needed from native English speaker to review the content.